src/plenoptic/synthesize/eigendistortion.py

```diff
@@ -69,38 +95,30 @@ def fisher_info_matrix_eigenvalue(
 
 
 class Eigendistortion(Synthesis):
-    r"""Synthesis object to compute eigendistortions induced by a model on a given
-    input image.
+    r"""
+    Synthesize eigendistortions induced by a model on a given input image.
+
+    Following the basic idea in [1]_, this class synthesizes image perturbations that
+    are considered the most and least noticeable for a model on a given image. Because
+    these are perturbations on the input image, they are local in pixel space, i.e.,
+    they do not change the pixels much.
 
     Parameters
     ----------
     image
-        Image. We currently do not support batches of images, as each image requires its
-        own optimization, so either ``image.ndimension()==1`` or ``image.shape[0]==1``
+        Image to perturb. We currently do not support batches of images, as each image
+        requires its own optimization, so either ``image.ndimension()==1`` or
+        ``image.shape[0]==1``.
     model
         Torch model with defined forward and backward operations.
 
-    Attributes
-    ----------
-    jacobian: Tensor
-        Is only set when :func:`synthesize` is run with ``method='exact'``. Default to
-        ``None``.
-    eigendistortions: Tensor
-        Tensor of eigendistortions (eigenvectors of Fisher matrix), ordered by
-        eigenvalue, with Size((n_distortions, *image.shape[1:])).
-    eigenvalues: Tensor
-        Tensor of eigenvalues corresponding to each eigendistortion, listed in
-        decreasing order.
-    eigenindex: listlike
-        Index of each eigenvector/eigenvalue.
-
     Notes
     -----
     This is a method for comparing image representations in terms of their ability to
-    explain perceptual sensitivity in humans. It estimates eigenvectors of the FIM.
-    A model, :math:`y = f(x)`, is a deterministic (and differentiable)
-    mapping from the input pixels :math:`x \in \mathbb{R}^n` to a mean output
-    response vector :math:`y\in \mathbb{R}^m`, where we assume additive white
+    explain perceptual sensitivity in humans. It estimates eigenvectors of the Fisher
+    Information Matrix. A model, :math:`y = f(x)`, is a deterministic (and
+    differentiable) mapping from the input pixels :math:`x \in \mathbb{R}^n` to a mean
+    output response vector :math:`y\in \mathbb{R}^m`, where we assume additive white
     Gaussian noise in the response space.
     The Jacobian matrix at x is:
```

Comment:
```suggestion
    The Jacobian matrix at :math:`x` is:
```
(if you want to be consistent with the "x" in line 131)
there's also an "f" in 126 and an "x" in 127 not in math mode.